src/libs/ReportUtils.js

```diff
@@ -1018,9 +1018,9 @@ function isUnread(report) {
         return false;
     }
 
-    const lastReadSequenceNumber = report.lastReadSequenceNumber || 0;
-    const maxSequenceNumber = report.maxSequenceNumber || 0;
-    return lastReadSequenceNumber < maxSequenceNumber;
+    const lastActionCreated = report.lastActionCreated || '';
+    const lastReadTime = report.lastReadTime || '';
+    return lastReadTime < lastActionCreated;
```

Comment:
NAB, sorry, I just keep forgetting.. what format and/or type are `lastReadTime` and `lastActionCreated`? They can be compared with a less than symbol? This might be a good thing to add a very brief comment.

Reply:
They are datetime strings, so they can be compared with comparison operators. I added a comment.